themes/10up-theme/assets/css/frontend/style.css

```diff
@@ -4,25 +4,33 @@
  *
  */
 
+/* Tailwind */
```

Comment:
An overall question for you @Nikki-Jones @iansvo - does `@apply` support nesting? 

```
.some-css-rule {
  @apply p-4;

  & .some-child-rule {
   @apply m-3;
  }
}
```

Reply:
@dainemawer I can confirm this functions out of the box: 

<img width="986" alt="image" src="https://user-images.githubusercontent.com/11017412/185142549-3bf97f72-8ac6-4609-8471-c456aa388218.png">